src/app/organizations/organization-details/manage-projects/manage-single-project/project-deposit/project-deposit.component.ts

```diff
@@ -49,4 +49,8 @@ export class ProjectDepositComponent {
         this.router.navigate(['../'], {relativeTo: this.route});
         return EMPTY;
     }
+
+    backToProjectEditScreen() {
+        this.router.navigate(['../../'], {relativeTo: this.route});
+    }
```

Comment:
Could we try to use routerLink for navigation? It is reasonable to use a method like this only if you are doing something along with navigating; let's say, if you are logging something and you are navigating.